docs/tutorials/T1_intro.ipynb

```diff
@@ -17,7 +17,7 @@
     "- Running a simulation\n",
     "- Plotting results\n",
     "\n",
-    "This tutorial walks you through how to define paramaters and run the simulation. The next tutorial will show you how to plot the results of a simulation."
+    "This tutorial walks you through how to define parameters and run the simulation. The next tutorial will show you how to plot the results of a simulation."
```

Comment:
Line 28: heading should be changed to something like "import FPsim"
Line 57-62: needs some punctuation or formatting
Line 64 typo 'these'
Line 148: can you add a description of what this line does?

Reply:
I'm having a hard time finding the line numbers in the notebook, so these are likely duplicate comments.
This line needs some clarity or formatting.  Not sure I understand the parameter categories as you lay them out
- Parameters are defined as a dictionary. In FPsim, we categorize our parameters as:
Basic parameters Age limits Durations Pregnancy outcomes Fecundity and exposure MCPR